src/pages/about-us/_OurPrinciples.tsx

```diff
@@ -0,0 +1,136 @@
+import React from 'react'
+import styled from 'styled-components'
+import { our_principles } from './_data'
+import { localize } from 'components/localization'
+import { SectionContainer, Flex, Box } from 'components/containers'
+import { Header, Divider } from 'components/elements'
+import device from 'themes/device'
+import { LinkButton } from 'components/form'
+
+const StyledSection = styled(SectionContainer)`
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    padding: 24px 16px 120px;
+    background-color: var(--color-white);
+
+    @media ${device.tablet} {
+        padding: 40px 16px;
+    }
+`
+const StyledHeader = styled(Header)`
+    margin: 0 0 40px 0;
+    padding: 0;
+    @media ${device.laptop} {
+        font-size: 28px;
+        margin-bottom: 24px;
+    }
+`
+const StyledDivider = styled(Divider)`
+    margin: 0;
+    @media (min-width: 824px) {
+        width: 792px;
+    }
+`
+
+const StyledFlex = styled(Flex)`
+    margin-bottom: 40px;
+    max-width: 792px;
+    @media (max-width: 930px) {
+        flex-direction: column;
+        width: unset;
+        align-items: left;
+        margin-bottom: 24px;
+    }
+`
+
+const InsideFlex = styled(Flex)`
+    @media ${device.tablet} {
+        display: block;
+        margin-top: 18px;
+    }
+`
+
+const StyledText = styled(Header)`
+    max-width: 551px;
+    margin: 16px 0;
+    font-size: 16px;
+`
+
+const StyledLinkButton = styled(LinkButton)`
+    height: 40px;
+    width: auto;
+    border-radius: 4px;
+    position: relative;
+    white-space: nowrap;
+`
```

Comment:
Is this a new button? If so, can we add it to our general component?

Reply:
I copied it from the main page.